website/docs/d/private_endpoint.html.markdown

```diff
@@ -0,0 +1,51 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: private_endpoint"
+sidebar_current: "docs-mongodbatlas-datasource-private-endpoint"
+description: |-
+    Describes a Private Endpoint.
+---
+
+# mongodbatlas_private_endpoint
+
+`mongodbatlas_private_endpoint` describe a Private Endpoint. This represents a Private Endpoint Connection that wants to retrieve details by ID in an Atlas project.
+
```

Comment:
Small edit, replace 2nd sentence with:
This represents a Private Endpoint Connection to retrieve details regarding a private endpoint by id in an Atlas project.

Reply:
Thanks @themantissa!